docs/source/conf.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# pytmpdir documentation build configuration file, created by
+# sphinx-quickstart on Wed Dec  7 16:24:30 2016.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+    'sphinx.ext.viewcode']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'pytmpdir'
+copyright = '2016, Synerty'
+author = 'Synerty'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '0.1.0'
```

Comment:
I have a pipbuild.sh script, You should copy it and also make it update this file https://github.com/Synerty/peek_worker/blob/master/pipbuild.sh

You should add a line like this to your pytmpdir/__init__.py for pipbuild to update https://github.com/Synerty/peek_worker/blob/master/peek_worker/__init__.py#L2

And it looks like this conf file is python, so you could just write (after the above is done)
import pytmpdir
version = pytmpdir.__version__
release = pytmpdir.__version__